drv/stm32h7-sprot-server/src/main.rs

```diff
@@ -1341,6 +1341,24 @@ impl<S: SpiServer> idl::InOrderSpRotImpl for ServerImpl<S> {
             Err(SprotProtocolError::UnexpectedResponse)?
         }
     }
+
+    fn policy_dev_or_release(
+        &mut self,
+        _msg: &userlib::RecvMessage,
+    ) -> Result<
+        drv_sprot_api::PolicyDevOrRelease,
+        idol_runtime::RequestError<SprotError>,
+    > {
+        let body = ReqBody::Policy(PolicyReq::DevOrRelease);
+        let tx_size = Request::pack(&body, self.tx_buf);
+        let rsp =
+            self.do_send_recv_retries(tx_size, TIMEOUT_LONG, DEFAULT_ATTEMPTS)?;
```

Comment:
**question:** Shouldn't the timeout here (probably) be `TIMEOUT_QUICK`? I'd imagine the reading of the data doesn't take that long. If memory serves, the `component_switch_default_image` function uses `TIMEOUT_LONG` because it may include rebooting the device (or something like that).

Reply:
Fixed in f10d7be9565c